jsonnetfile.jsonnet

```diff
@@ -1,6 +1,16 @@
 {
   version: 1,
   dependencies: std.prune([
+    {
+      source: {
+        git: {
+          remote: 'https://github.com/projectsyn/jsonnet-libs',
+          subdir: '',
+        },
+      },
+      version: 'initial-implementation',
```

Comment:
Do you want to wait until this branch is merged?

Reply:
Yes, see the TODO section in the description.